Example/Sources/View Controllers/MessageSubviewViewController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MessageSubviewViewController.swift
```

Comment:
This needs proper MIT License Header, copy that from `AppDelegate.swift` for example please :)